liquibase-core/src/main/java/liquibase/integration/spring/SpringResource.java

```diff
@@ -40,7 +41,7 @@ public liquibase.resource.Resource resolve(String other) {
     @Override
     public liquibase.resource.Resource resolveSibling(String other) {
         try {
-            Resource otherResource = this.resource.createRelative("../"+other);
+            UrlResource otherResource = new UrlResource(this.resource.createRelative(other).getURL());
```

Comment:
I don't think this is the correct fix. Switching an existing Resource implementation to UrlResouce class is just working around the problem that there are differences between the Resource implementations that shouldn't be there.

I'm thinking maybe #3355 is the more wide-ranging fix for this?